test/Microsoft.Health.Dicom.Web.Tests.E2E/Common/TestServerFeatureSettingType.cs

```diff
@@ -0,0 +1,27 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+
+namespace Microsoft.Health.Dicom.Web.Tests.E2E.Common;
+
+/// <summary>
```

Comment:
nit: remove the empty summary or add a summary.

Reply:
Done.